src/main/java/uk/gov/hmcts/darts/common/entity/HearingReportingRestrictionsEntity.java

```diff
@@ -56,7 +58,7 @@ public class HearingReportingRestrictionsEntity {
     OffsetDateTime eventDateTime;
 
     @Column(name = "case_number")
-    String caseNumber;
+    private List<String> caseIdList = new ArrayList<>();
```

Comment:
this should be called caseNumberList so as not to be confused with caseIds which are different